Tests/Resources/TestOperations/TypeScript/NodeInterfacesQuery.graphql

```diff
@@ -4,12 +4,20 @@ query NodeInterfaces($nodeId : ID!) {
     ...NodeId
     ... on Product{
         ... ProductNodeTitle
+        description
```

Comment:
Would it make sense to also include a case where we just have a fragment spread, like what we had tested before?